product/charts/layouts/daily_perf_charts/MiddlewareDatasource.yaml

```diff
@@ -0,0 +1,14 @@
+# Middleware Datasource daily performance chart layouts
+---
+- :title: Availability (Connections)
```

Comment:
Not sure I understand what Availability (Connections) means - availability of database connections via this datasource?